io.catenax.manufacturing_capability/1.0.0/ManufacturingCapability.ttl

```diff
@@ -0,0 +1,176 @@
+#######################################################################
+# Copyright (c) 2023 Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+#

```

Comment:
add 
# Copyright (c) 2022 Contributors to the Eclipse Foundation

for single contributions not covered by explicit mentions

Reply:
Done